src/snapshot/benches/version_map.rs

```diff
@@ -1,133 +0,0 @@
-// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Oh removing this will need a change in https://github.com/firecracker-microvm/firecracker/blob/main/tools/bump-version.sh to stop looking for versions in this file. I think it's the only one.

Reply:
Very nice catch. Can you think of another check to replace this with, or should we just drop the check all together?

Reply:
Also, is there a way to test the release scripts?

Reply:
We have a "release sanity build" step in our CI, but it only triggers if specific files in `tools` are touched. You could add a dummy commit that touches `tools/devtool` to trigger it

Reply:
I would go with dropping the check altogether. The issue that this was checking for is gone with your PR, so we don't need it anymore. You can test the release scripts locally by `devtool build --release`, but if you drop the check it will get done in the PR anyway.

Reply:
Ok, done

Reply:
it might be worth to run the check in every PR (?)